module_utils/network_lsr/nm_provider.py

```diff
@@ -21,3 +22,18 @@ def get_nm_ethtool_feature(name):
 
     feature = getattr(Util.NM(), name, None)
     return feature
+
+
+def get_nm_ethtool_setring(name):
+    """
+    Translate ethtool set-ring into Network Manager name
```

Comment:
`Translate ethtool set-ring option into NetworkManager attribute name` I think is a better description.

Reply:
Just `ring`

Reply:
Changed into `Translate ethtool ring option into NetworkManager attribute name`